src/Auth0.ManagementApi/Models/TenantFlags.cs

```diff
@@ -36,5 +36,95 @@ public class TenantFlags
+        /// Whether ID tokens can be used to authorize some types of requests to API v2 (true) not not (false).
+        /// </summary>
+        [JsonProperty("enable_idtoken_api2")]
+        public bool? EnableIdTokenApi2 { get; set; } = null;
+
+        /// <summary>
+        /// Whether the public sign up process shows a user_exists error (true) or a generic error (false) if the user already exists.
+        /// </summary>
+        [JsonProperty("enable_public_signup_user_exists_error")]
+        public bool? EnablePublicSignupUserExistsError { get; set; } = null;
+
+        /// <summary>
+        /// Whether the legacy delegation endpoint will be enabled for your account (true) or not available (false).
+        /// </summary>
+        [JsonProperty("allow_legacy_delegation_grant_types")]
+        public bool? AllowLegacyDelegationGrantTypes { get; set; } = null;
+
+        /// <summary>
+        /// Whether the legacy auth/ro endpoint (used with resource owner password and passwordless features) will be enabled for your account (true) or not available (false).
+        /// </summary>
+        [JsonProperty("allow_legacy_ro_grant_types")]
+        public bool? AllowLegacyRoGrantTypes { get; set; } = null;
+
+        /// <summary>
+        /// Whether users are prompted to confirm log in before SSO redirection (false) or are not prompted (true).
+        /// </summary>
+        [JsonProperty("enable_sso")]
+        public bool? EnableSSO { get; set; } = null;
+
+        /// <summary>
+        /// Do not Publish Enterprise Connections Information with IdP domains on the lock configuration file.
+        /// </summary>
+        [JsonProperty("no_disclose_enterprise_connections")]
+        public bool? NoDiscloseEnterpriseConnections { get; set; } = null;
+
+        /// <summary>
+        /// If true, SMS phone numbers will not be obfuscated in Management API GET calls.
+        /// </summary>
+        [JsonProperty("disable_management_api_sms_obfuscation")]
+        public bool? DisableManagementApiSmsObfuscation { get; set; } = null;
+
+        /// <summary>
+        /// Enforce client authentication for passwordless start
+        /// </summary>
+        [JsonProperty("enforce_client_authentication_on_passwordless_start")]
+        public bool? EnforceClientAuthenticationOnPasswordlessStart { get; set; } = null;
+
+        /// <summary>
+        /// Changes email_verified behavior for Azure AD/ADFS connections when enabled. Sets email_verified to false otherwise.
+        /// </summary>
+        [JsonProperty("trust_azure_adfs_email_verified_connection_property")]
+        public bool? TrustAzureAdfsEmailVerifiedConnectionProperty { get; set; } = null;
+
+        /// <summary>
+        /// Enables the email verification flow during login for Azure AD and ADFS connections
+        /// </summary>
+        [JsonProperty("enable_adfs_waad_email_verification")]
+        public bool? EnabdleAdfsWaadEmailVerification { get; set; } = null;
```

Comment:
There's a typo here, should it be `EnableAdfsWaadEmailVerification`?

Reply:
Wow, thanks! Great catch.